system/logs/templates/opentelemetry-logs-collector.yaml

```diff
@@ -114,32 +117,15 @@ spec:
           - action: insert
             key: failover_username_opensearch
             value: ${failover_username_b}
-      transform/ingress:
-        error_mode: ignore
-        log_statements:
-          - context: log
-            conditions:
-              - resource.attributes["app.label.name"] == "ingress-nginx"
-            statements:
-              - merge_maps(attributes, ExtractGrokPatterns(body, "%{IP:client.address} %{NOTSPACE:client.ident} %{NOTSPACE:client.auth} \\[%{HTTPDATE:timestamp}\\] \"%{WORD:request_method} %{NOTSPACE:request_path} %{WORD:network.protocol.name}/%{NOTSPACE:network.protocol.version}\" %{NUMBER:response} %{NUMBER:content_length:int} %{QUOTEDSTRING} \"%{GREEDYDATA:user_agent}\" %{NUMBER:request_length:int} %{BASE10NUM:request_time:float}( \\[%{NOTSPACE:service}\\])? ?(\\[\\])? %{IP:server.address}\\:%{NUMBER:server.port} %{NUMBER:upstream_response_length:int} %{BASE10NUM:upstream_response_time:float} %{NOTSPACE:upstream_status} %{NOTSPACE:request_id}", true),"upsert")
-              - set(attributes["network.protocol.name"], ConvertCase(attributes["network.protocol.name"], "lower")) where attributes["network.protocol.name"] != nil
-              - set(attributes["config.parsed"], "ingress-nginx") where attributes["client.address"] != nil
-
-      transform/files:
-        error_mode: ignore
-        log_statements:
-          - context: log
-            conditions:
-              - resource.attributes["log.type"] == "files"
-            statements:
-              - merge_maps(attributes, ExtractGrokPatterns(body, "%{TIMESTAMP_ISO8601:timestamp}%{SPACE}%{GREEDYDATA:log}", true), "upsert")
-              - set(attributes["config.parsed"], "files") where attributes["log_level"] != nil
 
 {{- if .Values.openTelemetryPlugin.openTelemetry.logsCollector.journaldConfig.enabled }}
     {{- include "journald.transform" . | nindent 6 -}}
 {{- end }}
-{{- if .Values.openTelemetryPlugin.openTelemetry.logsCollector.containerdConfig.enabled }}
+{{- if and (.Values.openTelemetryPlugin.openTelemetry.logsCollector.containerdConfig.enabled) (.Values.openTelemetryPlugin.openTelemetry.logsCollector.openstackConfig.enabled) }}
```

Comment:
What is this condition supposed to check on? Should there be anything in this block?

Reply:
we should have two configs, one for openstack and another one for containerd in general, both should not be active at the same time. That was my idea.

Reply:
it should exit the rendering, because both shouldn't be active at the same time.